GTBitsOfGood/angels-among-us
# access management page

## access management page

Issue Number(s): #21.

What does this PR change and why?
Update access management page to accommodate 3 user types & new design. Implement remove accounts / edit permissions feature.

### Checklist
 
- [x] Follows Figma design
- [x] Responsive - desktop and mobile
- [x] Input validation still works

### Critical Changes

### Related PRs
#5 

### Testing

1. Use create account form to input valid email address and verify that account data populates correctly in the top row of the table.
2. Use create account form to input an invalid email address. Check that the user is alerted. Correct the email and resubmit the form. The error should disappear upon successful submission and the new account data populates first row of table.
3. In select mode user can select / deselect accounts then cancel or proceed to delete. Popup prompting delete functions as expected. 
4. Popups close when the user clicks outside.
5. Changing an account's permission appears as expected in subsequent re-renders.
